src/ui/templates/overlay_iframe.html

```diff
@@ -21,13 +21,20 @@
   <link rel="stylesheet" href="./node_modules/material-design-lite/material.min.css">
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <script src="./node_modules/material-design-lite/material.min.js"></script>
-  <script src="./node_modules/react/dist/react.js"></script>
-  <script src="./node_modules/react-dom/dist/react-dom.js"></script>
+  <script src="./node_modules/react/dist/react.min.js"></script>
+  <script src="./node_modules/react-dom/dist/react-dom.min.js"></script>
 </head>
 
 <body>
   <div id="rmx-overlay-wrapper"></div>
-  <script src="./node_modules/material-remixer/dist/overlay.js"></script>
+  <script>
+    // Dynamically load the overlay(.min?).js from same path as remixer script.
+    var remixerPath = window.parent.document.querySelector('script[src*="remixer"]');
+    var overlayPath = remixerPath.src.replace('remixer', 'overlay');
```

Comment:
The `remixer.js` file is actually responsible for loading this .html file, so it should definitely not be null. However if for some strange reason is, its a no-op and should display console error with missing filename, which I think will be useful to the client as to why to overlay didn't show up.